Algorithm/QCAlgorithm.Indicators.cs

```diff
@@ -223,6 +223,38 @@ public RelativeStrengthIndex RSI(string symbol, int period, MovingAverageType mo
         }
 
         /// <summary>
+        /// Creates a new StandardDeviation indicator. This will return the population standard deviation of samples over the specified period.
+        /// </summary>
+        /// <param name="symbol">The symbol whose STD we want</param>
+        /// <param name="period">The period over which to compute the STD</param>
+        /// <param name="resolution">The resolution</param>
+        /// <returns>The StandardDeviation indicator for the requested symbol over the speified period</returns>
+        public StandardDeviation STD(string symbol, int period, Resolution? resolution = null)
+        {
+            var name = CreateIndicatorName(symbol, "STD" + period, resolution);
+            var std = new StandardDeviation(name, period);
+            RegisterIndicator(symbol, std, resolution, x => x.Value);
+            return std;
+        }
+
+        /// <summary>
+        /// Creates a new BollingerBands indicator which will compute the MiddleBand, UpperBand, LowerBand, and StandardDeviation
+        /// </summary>
+        /// <param name="symbol">The symbol whose BollingerBands we seek</param>
+        /// <param name="period">The period of the standard deviation and moving average (middle band)</param>
+        /// <param name="k">The number of standard deviations specifying the distance between the middle band and upper or lower bands</param>
+        /// <param name="movingAverageType">The type of moving average to be used</param>
+        /// <param name="resolution">The resolution</param>
+        /// <returns>A BollingerBands configured with the specied period</returns>
+        public BollingerBands BOL(string symbol, int period, int k, MovingAverageType movingAverageType = MovingAverageType.Simple, Resolution? resolution = null)
```

Comment:
I think we'd like to use BB for the short code helper function here, also, the ability type specify 'k' as a decimal would be nice too, this would allow me to use 1/2 a std deviation instead of full integer incremements